app/_locales/en/messages.json

```diff
@@ -5159,6 +5159,15 @@
   "somethingWentWrong": {
     "message": "Oops! Something went wrong."
   },
+  "sortBy": {
+    "message": "Sort By"
+  },
+  "sortByAlphabetically": {
+    "message": "Alphabetically (A-Z)"
+  },
+  "sortByDecliningBalance": {
+    "message": "Declining balance ($ high-low)"
```

Comment:
We might want to localize the currency unit if possible e.g. show € if that's their preferred currency

Reply:
the messages can be templated with `$1`.  Question is whether there's a convenient way to get e.g. € from `EUR`. Some probably don't have a single symbol, especially other cryptos, and would need to be written out  like `ZEC` for zcash

Reply:
We do something like this on mobile. I can copy that pattern over.

Reply:
https://github.com/MetaMask/metamask-extension/pull/27184/commits/bda74e935396c157badea68508d89a5334f60ad8